components/observations/ObservationsListView.tsx

```diff
@@ -151,27 +157,33 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
         return;
       }
 
+      // Date limit is set to 6 months in the past, 1.5552e10 is 180 days in milliseconds
```

Comment:
The one thing to be wary of is pre-season testing (looking at the app in November, often folks are doing this to validate things before the season starts and need to see the latest data which could be from April). Let's set this to ~1y or something and use the `date-fns` [helper](https://date-fns.org/v4.1.0/docs/sub) instead of magic numbers.